pyseir/deployment/demographic_mapper.py

```diff
@@ -0,0 +1,719 @@
+            HR = hospital_inflow_rates
+            for key in HR:
+                IHR[key][measure_unit.value] = HR[key][:, np.newaxis] * fraction_of_symptomatic
+
+        elif measure_unit is CovidMeasureUnit.PER_CAPITA:
+            total_rate_out_of_I = self.parameters['delta']
+            for key in hospital_inflow_rates:
+                total_rate_out_of_I += hospital_inflow_rates[key]
+
+            HR = {}
+            for key in hospital_inflow_rates:
+                HR[key] = np.tile(hospital_inflow_rates[key] / total_rate_out_of_I,
+                                  (len(self.parameters['t_list']), 1)).T
+            for key in HR:
+                IHR[key][measure_unit.value] = HR[key] * fraction_of_symptomatic
+
+        return IHR
+
+    def _age_specific_IFR_by_hospitalization(self, measure_unit, IHR):
+        """
+        Calculates age specific mortality with given unit among infected
+        population (IFR) by types of hospitalization.
+
+        When unit is per capita, mortality rate is calculated as the
+        probability that an infected person dies of covid-19:
+            probability of hospitalization * mortality rate /
+            (mortality rate + recovery rate)
+
+        When unit is per capita day, mortality rate is calculated as
+        probability of dying of covid-19 per capita per day among infected
+        population:
+            new death / (asymptomatic + symptomatic + hospitalized)
+
+        Parameters
+        ----------
+        measure_unit: CovidMeasureUnit
+            Unit of the covid measure.
+        IHR: dict
+            Rates of hospitalization by age group, type of hospitalization,
+            and unit of rates, with type of hospitalization as primary key,
+            unit as secondary key, and array of corresponding time series of
+            rates as values.
+            For example, IHR['HGen']['per_capita'] is the
+            time series of probability of being admitted to non-ICU per
+            capita among infected population (asymptomatic + symptomatic).
+
+        Returns
+        -------
+        IFR_general: np.array
+            Rates of mortality in non-ICU of infected population
+        IFR_icu: np.array
+            Rates of mortality in ICU of infected population
+        IFR_icu_vent: np.array
+            Rates of mortality in ICU with Ventilator of infected population
+        """
+        mortality_inflow_rates = self._reconstruct_mortality_inflow_rates()
+
+        hospital_recovery_inflow_rates = self._reconstruct_hospital_recovery_inflow_rates()
+
+        mortality_probs = {}
+        for key in hospital_recovery_inflow_rates:
+            mortality_probs[key] = \
+                mortality_inflow_rates[key] / (mortality_inflow_rates[key] + hospital_recovery_inflow_rates[key])
+
+        IFR = defaultdict(dict)
+
+        for key in mortality_probs:
+            IFR[key][measure_unit.value] = mortality_probs[key] * IHR[key][measure_unit.value]
```

Comment:
Could you add an `if measure_unit is CovidMeasureUnit.PER_CAPITA` statement in the appropriate place here to make things more clear?